packages/app-store/exchangecalendar/lib/CalendarService.ts

```diff
@@ -62,11 +63,11 @@ export default class ExchangeCalendarService implements Calendar {
     appointment.Location = event.location || "";
     appointment.Body = new MessageBody(event.description || "");
     event.attendees.forEach((attendee: Person) => {
-      appointment.RequiredAttendees.Add(new Attendee(attendee.email));
+      appointment.RequiredAttendees.Add(new Attendee(attendee.email ?? BOOKED_WITH_SMS_EMAIL));
```

Comment:
Why is the BOOKED_WITH_SMS_EMAIL necessary?

Reply:
email is required field here to create an event and BOOKED_WITH_SMS would also help us know that it was booked with phone number.